cmd/lncli/commands.go

```diff
@@ -2810,20 +2810,52 @@ func decodePayReq(ctx *cli.Context) error {
 	return nil
 }
 
+const defaultListChainTxnsLimit int = 1000
```

Comment:
Instead of defining the limit here, it could be done within the `cli.IntFlag`. See the `Value` field.

Reply:
Thanks. fixed.